assets/clients/clients.yaml

```diff
@@ -47,6 +49,7 @@ clients:
   - name: "Jellyfin for Android"
     targets: [ Android ]
     oss: https://github.com/jellyfin/jellyfin-android
+    official: false
```

Comment:
Isn't **Jellyfin for Android** official? On https://jellyfin.org/downloads/ it has the ` Official ` badge

Reply:
Yes it is, sorry must have mixed there something up